common/logging/src/lib.rs

```diff
@@ -214,6 +224,56 @@ impl TimeLatch {
     }
 }
 
+pub fn create_tracing_layer(logfile_max_size: u64, base_tracing_log_path: PathBuf) {
+    let filter_layer = match tracing_subscriber::EnvFilter::try_from_default_env()
+        .or_else(|_| tracing_subscriber::EnvFilter::try_new("warn"))
+    {
+        Ok(filter) => filter,
+        Err(e) => {
+            eprintln!("Failed to initialize dependency tracing {e}");
```

Comment:
I think at this point we don't have a useable slogger, so we are forced into prtinln! right?

Reply:
yep that's the reason